migrate/20240529_github_cache.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:github_cache_entry) do
```

Comment:
Have you estimated very roughly how much traffic we'd route through our control plane and how many queries we'd make to Postgres if say 25% of our GH runners customers start to use our cache?

Reply:
(_Sorry for the delayed reply. I thought I had already responded to this comment, but it seems I forgot to post it_)

Actually, I haven't done it yet, but let's do it here. I expect it will be very lower than our current traffic. GitHub sends all customer webhook events to our web server, regardless of whether they use ubicloud in their workflow or not. This results in high traffic on our web server. Usually, customer workflows are set up for a single language in their job, which means they download only one cache file. If the cache exists, it requires just one web request. If the cache doesn't exist, it requires two requests: reserve and commit. We use a single table for cache lookups, so the queries aren't complex. If we encounter performance issues, we can add indices to some columns.